docs/releases/4.2.txt

```diff
@@ -483,6 +483,9 @@ Miscellaneous
 * The undocumented ``negated`` parameter of the
   :class:`~django.db.models.Exists` expression is removed.
 
+* Add functionality so :class:`~django.db.models.sql.RawQuery`
+  can be used as a subquery.
```

Comment:
`RawQuery` is not part of the public API. I think instead we can add a new section to https://docs.djangoproject.com/en/5.2/topics/db/sql/#performing-raw-queries which says that the raw queries can be used in subquerys (with a `.. versionadded:: 6.0` note) then we can add a release note in the 6.0 release notes that refer to this